lib/players/computer.rb

```diff
@@ -0,0 +1,139 @@
+class Computer < Player
+
+  def move(board)
```

Comment:
This method is reallllllly long. Consider breaking it up into some smaller chunks.